cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -110,7 +110,7 @@ type CMServer struct {
 func NewCMServer() *CMServer {
 	s := CMServer{
 		Port:                              ports.ControllerManagerPort,
-		Address:                           net.ParseIP("127.0.0.1"),
+		Address:                           net.ParseIP("0.0.0.0"),
```

Comment:
...maybe we should conduct a survey of what this binary serves before potentially serving it to the world. If it's all read-only things, I think this should be OK?